catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -174,7 +175,6 @@
                             solr-core,
                             solr-solrj,
                             solr-factory,
-                            commons-io,
                             commons-lang,
                             joda-time,
                             lucene-core,
```

Comment:
One of the errors I received when testing the search ui disappeared after adding lucene-join to just catalog-solr-embedded-provider's pom, but I think it's a good idea to add it those other poms just in case.